docker-compose.yml

```diff
@@ -191,6 +191,19 @@ services:
         target: /usr/local/etc/redis
         read_only: False
 
+  flower:
+    image: mher/flower:2.0
+    command: celery flower --broker=redis://:redispassword@redis:6379/0
+    ports:
+      - "5555:5555"
+    depends_on:
+      - redis
+      - worker
+    environment:
+      - FLOWER_BASIC_AUTH=admin:adminpassword # Optional: basic auth protection
```

Comment:
If it's optional and the password in a public repo, is there any sense in having the authentication at all if it's just local?